.gitignore

```diff
@@ -28,4 +28,5 @@ tests.out
 bazel-bin
 bazel-manzan
 bazel-out
-bazel-testlogs
\ No newline at end of file
+bazel-testlogs
+MODULE.bazel.lock
```

Comment:
This file should be included in source control for consistent dependency resolution